### ISSUE TITLE
Breaking change for Relay connections between 1.1.0 - 2.1.0? Receiving multiple types named "PageInfo" on Schema generation

### ISSUE THREAD

**Issue by rgovindji** [mickhansen/graphql-sequelize#265]
My current package.json has the following dependencies:
```
  "dependencies": {
    "babel-relay-plugin": "^0.8.0",
    "express": "^4.13.4",
    "express-graphql": "^0.5.1",
    "express-jwt": "^3.3.0",
    "graphql": "^0.5.0",
    "graphql-relay": "^0.4.1",
    "graphql-sequelize": "^1.1.0",
    "gulp": "^3.9.1",
    "gulp-graphql": "^0.2.0",
    "pg": "^4.4.4",
    "pg-hstore": "^2.3.2",
    "react": "^15.0.1",
    "react-relay": "^0.8.0",
}
```

When I updated from "graphql-sequelize": "^1.1.0" -> "graphql-sequelize": "^2.1.0" and ran "npm run update-schema" I get the following error:

```
/myProject/node_modules/graphql/jsutils/invariant.js:19
    throw new Error(message);
    ^

Error: Schema must contain unique named types but contains multiple types named "PageInfo".
    at invariant (/myProject/node_modules/graphql/jsutils/invariant.js:19:11)
    at typeMapReducer (/myProject/node_modules/graphql/type/schema.js:207:29)
    at typeMapReducer (/myProject/node_modules/graphql/type/schema.js:204:12)
    at /myProject/node_modules/graphql/type/schema.js:234:22
    at Array.forEach (native)
    at /myProject/node_modules/graphql/type/schema.js:225:37
    at typeMapReducer (/myProject/node_modules/graphql/type/schema.js:236:7)
    at /myProject/node_modules/graphql/type/schema.js:234:22
    at Array.forEach (native)
    at /myProject/node_modules/graphql/type/schema.js:225:37
    at typeMapReducer (/myProject/node_modules/graphql/type/schema.js:236:7)
    at /myProject/node_modules/graphql/type/schema.js:234:22
    at Array.forEach (native)
    at /myProject/node_modules/graphql/type/schema.js:225:37
    at typeMapReducer (/myProject/node_modules/graphql/type/schema.js:236:7)
    at Array.reduce (native)
    at new GraphQLSchema (/myProject/node_modules/graphql/type/schema.js:104:34)
    at Object.<anonymous> (schema.js:2875:16)
    at Module._compile (module.js:413:34)
    at loader (/myProject/node_modules/babel-cli/node_modules/babel-register/lib/node.js:158:5)
    at Object.require.extensions.(anonymous function) [as .js] (/myProject/node_modules/babel-cli/node_modules/babel-register/lib/node.js:168:7)
    at Module.load (module.js:357:32)

```

**Comment by brad-decker** [comment]
I haven't yet updated to the 2.x version. @mickhansen i'll test this at work and see if i get similar results and report back. Thanks @rgovindji

**Comment by mickhansen** [comment]
This is usually the result of having multiple instances of graphql-relay, try making sure you have the same version as graphql-sequelize uses (will be fixed in the next version where we switch to peer dependency)

**Comment by rgovindji** [comment]
@mickhansen thanks for the fix, updated to latest version v2.1.1 and the issue is resolved
